docs/src/main/asciidoc/security-built-in-authentication.adoc

```diff
@@ -28,7 +28,7 @@ be read by all members of the cluster (provided they all share the same encrypti
 The encryption key can be set using the `quarkus.http.auth.session.encryption-key` property, and it must be at least 16 characters
 long. This key is hashed using SHA-256 and the resulting digest is used as a key for AES-256 encryption of the cookie
 value. This cookie contains an expiry time as part of the encrypted value, so all nodes in the cluster must have their
-clocks synchronised. At one minute intervals a new cookie will be generated with an updated expiry time if the session
+clocks synchronized. At one minute intervals a new cookie will be generated with an updated expiry time if the session
```

Comment:
Ah ah, so these ones (and the `behaviour` one) are not actually typos but "traditional" English from UK compared to US English.

That being said, given US English is more popular these days and you also fixed real typos, I would be in favor of merging it.

Reply:
Yes indeed it's not always typos. 
@gsmet I have one question about homogeneity of the documentation (US vs UK words or quarkus vs Quarkus).  I can provide some PR about this. Are you ok with that?
If so, should I title each PR as `Documentation homogeneity`.